packages/mobile/src/components/scrubber/usePosition.ts

```diff
@@ -11,6 +12,7 @@ export const usePosition = (
 ) => {
   const positionRef = useRef(0)
   const positionElementRef = useRef<TextInput>(null)
+  const progress = useProgress(200)
```

Comment:
so the main idea here was to avoid using useProgress cause it causes rerender every 200 ms! lmk how you want to handle this, im thinking best way is to just update the setTimeout call to trigger depending on rate, it was 1000 for 1 second, but we can make it 15000 etc for whatever the playback rate is (would require a bit of math to relate to rate)

Reply:
thinking about this one, wouldn't updating the timeout to use a faster rate cause rerenders at that faster rate anyway? I can update the timeout function to read from TrackPlayer at its own rate and then update the rate to match the playbackRate, but want to make sure that it would actually save on rerenders